src/actions/teamActions.js

```diff
@@ -0,0 +1,24 @@
+import { LOAD_TEAMS_SUCCESS, TOGGLE_TEAM } from './actionTypes';
+import teamApi from '../api/teamApi';
+
+export function loadTeams() {
+  return function(dispatch) {
+    return teamApi.getAllTeams().then(teams => {
+      const teamsArray = teams.map(team => ({
+        ...team, visible: true
+      }));
```

Comment:
I am not a fan of creating variables when you don't need to.
You can write a function like `parseTeams`
```js
const parseTeams = teams => teams.map(team => ({ ...team, visible: true })
```
and use it:
```js
dispatch(loadTeamsSuccess(parseTeams(teams)))
```

or you can have it in the reducer, so you pass the teams as it is and add the visible in the reducer which I think would be the best solution :)